internal/commands/scan.go

```diff
@@ -95,9 +95,8 @@ const (
 		"\nTo use this feature, you would need to purchase a license." +
 		"\nPlease contact our support team for assistance if you believe you have already purchased a license." +
 		"\nLicensed packages: %s"
-	completedPolicy  = "COMPLETED"
```

Comment:
why did you removed this params?

Reply:
Unused variables, Got errors from GitHub linter